### DISCUSSION TITLE
Add support for more users / authors in notifications

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-future#374] [Feature Requests]
One user has requested the ability to notify the author of a post when it expires

**Comment by andergmartins** [comment]
@stevejburge Maybe this is an opportunity to implement an integration to PublishPress' notifications workflow? We have a good basis for that on PublishPress.

**Comment by stevejburge** [comment]
Yes, that might be the best option here rather than spending too much time with the basic integration
